README.md

```diff
@@ -8,9 +8,9 @@
 [npm-url]: https://www.npmjs.com/package/ember-frost-css-core
 
 # ember-frost-css-core <br /> [![Travis][ci-img]][ci-url] [![Coveralls][cov-img]][cov-url] [![NPM][npm-img]][npm-url]
-SASS based normalization, convenience classes and variables for Frost projects
+SASS based normalization, flex convenience classes and variables for Frost projects
```

Comment:
We may have more convenience than just flex in the future - this is just a short description, so I'd leave it out

We can probably drop "and variables" though, since they're all in theme now